src/web/components/AdSlot.tsx

```diff
@@ -20,6 +20,9 @@ export const labelStyles = css`
         color: ${text.supporting};
         text-align: left;
         box-sizing: border-box;
+        @media print{
+        display: none !important;
+        }
```

Comment:
I am relucatnt to use `!important` in general (maybe we can check that page for print and just not render?) also I do not know if we want to remove adds for print layout? @gtrufitt @oliverlloyd @jfsoul thoughts?

Reply:
You use `data-print-layout='hide'` in a lot of HTML to hide in print. Maybe that could be a better idea than using CSS `!important` ?